### PR TITLE
Added more native pseudo classes into nativePseudoClasses

### PR DESCRIPTION
**Pull request by wtfil**

### CODE REVIEW

**Comment by idoros** [comment]
We're approving this PR despite `focus-within` and `focus-visible` still being a draft. Make sure you polyfill as needed.

File: packages/core/src/native-reserved-lists.ts [hunk 1]
```diff
@@ -85,7 +87,8 @@ export const reservedKeyFrames = [
     'both',
     /* single-animation-play-state */
     'running',
-    'paused'
+    'paused',
+    'defined'
```
Comment (idoros) [inline packages/core/src/native-reserved-lists.ts]: Why is this under `reservedKeyFrames` and not `nativePseudoClasses`?

**Comment by wtfil** [comment]
@idoros i have moved `defined` into `nativePseudoClasses`

**Comment by wtfil** [comment]
@idoros Hi! Can you please merge?

**Comment by tomrav** [comment]
Merged, thanks for the help. :)

**Pull request merged**